pycm/pycm_compare.py

```diff
@@ -236,18 +238,19 @@ def __compare_assign_handler__(compare, cm_dict, weight, digit):
     if len(cm_dict) < 2:
         raise pycmCompareError(COMPARE_NUMBER_ERROR)
     compare.classes = list(cm_dict.values())[0].classes
-    compare.weight = {k: 1 for k in compare.classes}
+    compare.class_weight = {k: 1 for k in compare.classes}
     compare.digit = digit
     compare.best = None
     compare.best_name = None
     compare.sorted = None
     compare.scores = {k: {"overall": 0, "class": 0}.copy()
                       for k in cm_dict.keys()}
-    if weight is not None:
-        if not isinstance(weight, dict):
-            raise pycmCompareError(COMPARE_WEIGHT_ERROR)
-        if set(weight.keys()) == set(compare.classes) and all(
-                [isfloat(x) for x in weight.values()]):
-            compare.weight = weight
+    if class_weight is not None:
+        if not isinstance(class_weight, dict):
+            raise pycmCompareError(COMPARE_CLASS_WEIGHT_ERROR)
+        if set(class_weight.keys()) == set(compare.classes):
+            if all([isfloat(x) for x in class_weight.values()]
```

Comment:
Is a negative weight meaningful? 
If not, I think it would be good if here we can check if all of the weights are non-negative floats.

Reply:
No, it's not.
But I think this type of validation is out of our scope (We can talk more about it in the weekly meeting)